### PR TITLE
update python and swift example

### PR DESCRIPTION
**Pull request by ubbabeck**
What is done

moved the connecting to LSP from page 6 to page page 4 since I believe it makes more sense as many of the examples depend on `lsp_info`params. 

Updated the examples to work with the upcoming release of the sdk.

I have not updated the examples for fee calculation, will add this later.

### CODE REVIEW

File: src/guide/payments.md [hunk 1]
```diff
@@ -169,7 +170,7 @@ try {
 <section>
 
 ```python
-bolt11 = "..."
+bolt11 = receive_payment_response.ln_invoice.bolt11
```
[the remark this answers is not in the document]
Comment (ubbabeck) [reply src/guide/payments.md:173]: good point, tried to find a smart way to show of how to get the different values, haha. I'll change it back

File: src/guide/buy_btc.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+</section>
+
+<div slot="title">Swift</div>
+<section>
+
+```swift
+do {
+  let openingFeeParam:OpeningFeeParams? = nil
+  let req = BuyBitcoinRequest(
+    provider: .moonpay, openingFeeParams: openingFeeParam)
```
Comment (roeierez) [inline src/guide/buy_btc.md:23]: Do we have to pass the openingFeePrams? Isn't the swift use the default value?

File: src/guide/receive_onchain.md [hunk 3]
```diff
@@ -24,11 +25,9 @@ let address = swap_info.bitcoin_address;
 
 ```swift
 do {
-  // Optional user-selected dynamic fees (see Connecting to LSP section for details)
-  let opening_fee_params = nil;
-  let request = ReceiveOnchainRequest(opening_fee_params: opening_fee_params);
-
-  let swapInfo = try sdk.receiveOnchain(req: request)
+  let openingFeeParams:OpeningFeeParams? = nil
+  let req = ReceiveOnchainRequest(openingFeeParams: openingFeeParams)
```
Comment (roeierez) [inline src/guide/receive_onchain.md:29]: Can we omit the openingFeeParams from the request? We do have default value in the udl file for that.

**Review by roeierez — CHANGES_REQUESTED** [review]
Thanks Ruben.
I dropped some comments. My own preference is to use one line when possible instead of splitting the request and the actual call, e.g:

`sdk_services.receive_onchain(ReceiveOnchainRequest())`

But this is a matter of taste I guess.

File: src/guide/send_onchain.md [hunk 4]
```diff
@@ -290,9 +296,10 @@ try {
 ```python
 destination_address = "bc1.."
 amount_sat = current_fees.min
+fee_hash = current_fees.fee_hash
 
 try:
-  sdk.send_onchain(amount_sat, destination_address, current_fees.fees_hash)
+  sdk_services.send_onchain(amount_sat=amount_msats, onchain_recipient_address="...", pair_hash=fee_hash)
```
Comment (JssDWt) [inline src/guide/send_onchain.md:302]: Is this missing the sat_per_vbyte parameter?

File: src/guide/buy_btc.md (hunk 2 above)
Comment (ubbabeck) [reply src/guide/buy_btc.md]: Yes it does. Updated

File: src/guide/receive_onchain.md (hunk 3 above)
Comment (ubbabeck) [reply src/guide/receive_onchain.md]: Updated

File: src/guide/send_onchain.md [hunk 5]
```diff
@@ -21,12 +21,15 @@ info!("Estimated miner fees in sats for claiming funds: {}", current_fees.fees_c
...
+let sendAmountSat:UInt64? = nil
 try {
-  let currentFees = try sdk.fetchReverseSwapFees()
-
-  println("Percentage fee for the reverse swap service: \(currentFees.feesPercentage)")
-  println("Estimated miner fees in sats for locking up funds: \(currentFees.feesLockup)")
-  println("Estimated miner fees in sats for claiming funds: \(currentFees.feesClaim)")
+ let currentFees = try sdk.fetchReverseSwapFees(
+      req: ReverseSwapFeesRequest(sendAmountSat: sendAmountSat))
+  print("Percentage fee for the reverse swap service: \(currentFees.feesPercentage)")
```
[the remark this answers is not in the document]
Comment (ubbabeck) [reply src/guide/send_onchain.md]: Good point ! updated

File: src/guide/send_onchain.md (hunk 4 above)
Comment (ubbabeck) [reply src/guide/send_onchain.md]: Good catch, thanks! Updated.

**Review by roeierez — APPROVED** [review]
LGTM!

**Pull request merged**